### ISSUE TITLE
merge with openFrameworks core

### ISSUE THREAD

**Issue by ofZach** [openframeworks/projectGenerator#83]
@ofTheo reported that it's still not super clear with the project generator how to make examples for OF when you pull from github, I feel like moving this submodule back into the core of OF would help make things more clear and easier to track the progress of this project.     it's also been tricky to have PRs across both the main OF repo and this repo when things like templates and the addon change, as well as sometimes the submodule link is out of date.  

I'm in favor of bringing this back into the core of OF and making it a bit easier to understand and use the command line tool for updating the OF examples.

**Comment by HalfdanJ** [comment]
Something that would really help is a overview readme of this repo. I'm still confused on the different modules in the root of this repo.

**Comment by HalfdanJ** [comment]
Also based on personal experience, i think this is mostly about having a good readme in openframeworks root that describes properly how to set up the development enviroment for openframeworks, including what commands to get going with generating the examples.

**Comment by ofTheo** [comment]
would love this! 
It took @nickhardeman and I a couple of hours, including installing different versions of Git command line tools to get the submodule working.  My feeling was if it was this hard for us to do it, how many other people would have gotten frustrated and given up. 

I think a project generator label in the OF repo is all thats needed for filtering the PG issues. 
Also I think now that the big push has been made on PG - it won't really need a separate repo. 

Also at this point could we also remove projectGeneratorLegacy and Simple?

**Comment by ofZach** [comment]
yep on it -- https://github.com/openframeworks/projectGenerator/issues/82

I wonder if we could have a specific readme for developers and a readme for end users....  The developer readme would not be in the packaged release, but would give very clear info about how to setup the dev environment, etc...  the end user readme could be more friendly like where to ask questions, ask for help, etc.    It could also be more exhaustive and link back to the website docs which would be more up to date.

**Comment by HalfdanJ** [comment]
I think that would be great. Would be great if the readme on github was more focused on oF development then usage, and just pointing "end users" in the right direction (openframeworks.cc)

**Comment by elliotwoods** [comment]
i've hit into this a couple of times
finally realised the best way for me is to copy it out of the downloadable oF zip 
(i.e. i want a working version of oF from git, but don't want to deal with node.js)
kinda feels dirty, so just putting it out there so others don't feel bad for doing the same :)

**Comment by mattfelsen** [comment]
@elliotwoods jaw saw Arturo mention this over on #104...you can get a pg binary built from github from http://ci.openframeworks.cc/projectGenerator/ without having to download the whole OF release/nightly :+1:

**Comment by mattfelsen** [comment]
Er scratch that, just downloaded it and realized it's only the command line tool, not the packaged electron app with the bundled command line tool

**Comment by arturoc** [comment]
oh yeah sorry i also thought it was the whole thing when i posted it. i'll try to upload the latest tool somewhere too so there's an individual download for people working from git

**Comment by mattfelsen** [comment]
Err I referenced the wrong commit in my PR, meant to close #82 and not this one. Sorry!!
